Code/GraphMol/FileParsers/MolFileWriter.cpp

```diff
@@ -1382,7 +1383,7 @@ std::string MolToMolBlock(const ROMol &mol, bool includeStereo, int confId,
   if (trwmol.needsUpdatePropertyCache()) {
     trwmol.updatePropertyCache(false);
   }
-  if (kekulize) {
+  if (kekulize && mol.getNumBonds()) {
```

Comment:
Is this related to this PR?

Reply:
yes: the tests include a molecule with a single aromatic atom. That led to errors when generating a CTAB

Reply:
Nope, that's intended to fail if you have a molecule with aromatic atoms and no bonds: that clearly cannot be kekulized.

Reply:
cool